src/sentry/integrations/github/integration.py

```diff
@@ -98,6 +99,25 @@ class GitHubIntegrationProvider(IntegrationProvider):
         'height': 1000,
     }
 
+    def post_install(self, integration, organization):
+        client = GitHubAppsClient(integration=integration)
+
+        accessible_repos = client.get_repositories()
+        accessible_repo_names = [r['full_name'] for r in accessible_repos]
```

Comment:
This seems to have changed https://github.com/getsentry/sentry/commit/7ffc61b8b475aa706d17f8d098e4ec7a2e8600a7#diff-b42e9d7a44398599dab80a5a847ae6b9R62

Reply:
That's on `GitHubIntegration`, this is using `GitHubApiClient#get_repositories`. I'm not totally sure why we're duplicating it to format the data differently, but :man_shrugging: